src/Tizen.NUI/src/public/Utility/Capture.cs

```diff
@@ -152,9 +152,12 @@ public void Start(Container source, Position position, Size size, string path, C
 
             if (source is View || source is Layer)
             {
-                Interop.Capture.Start4(SwigCPtr, source.SwigCPtr, new Vector2(position.X, position.Y).SwigCPtr, new Vector2(size.Width, size.Height).SwigCPtr, path, new Vector4(color.R, color.G, color.B, color.A).SwigCPtr);
-
-                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                ReusablePool<Vector4>.GetOne((vector4, color) =>
+                {
+                    vector4.Reset(color.R, color.G, color.B, color.A);
+                    Interop.Capture.Start4(SwigCPtr, source.SwigCPtr, new Vector2(position.X, position.Y).SwigCPtr, new Vector2(size.Width, size.Height).SwigCPtr, path, vector4.SwigCPtr);
+                    NDalicPINVOKE.ThrowExceptionIfExists();
+                }, color);
```

Comment:
Actually, since the native handle of Color is Vector4, it would be more efficient to pass the SwigCPtr of color directly here.
```suggestion
                Interop.Capture.Start4(SwigCPtr, source.SwigCPtr, new Vector2(position.X, position.Y).SwigCPtr, new Vector2(size.Width, size.Height).SwigCPtr, path, color.SwigCPtr);
```

``` c#
        public Color(float r, float g, float b, float a) : this(Interop.Vector4.NewVector4(ValueCheck(r), ValueCheck(g), ValueCheck(b), ValueCheck(a)), true)
        {
            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
        }
```